tests/dns/dns-rcode/test.rules

```diff
@@ -0,0 +1,11 @@
+# Should alert in both directions as no flow is provided.
+alert dns any any -> any any (dns.rcode:3; sid:1; rev:1;)
```

Comment:
This should only alert in one direction, the reply. As the request has an rcode of 0.